addons/skin.confluence/720p/IncludesHomeRecentlyAdded.xml

```diff
@@ -140,7 +140,7 @@
 					<content>
 						<item>
 							<label>$INFO[Window.Property(LatestMovie.1.Title)]</label>
-							<label2/>
+							<label2 />
```

Comment:
...in this specific case, we could remove those lines completely. no need to define empty InfoLabels :)